bower.json

```diff
@@ -7,6 +7,7 @@
     "html5shiv": "~3.7.3",
     "jquery": "~2.1.4",
     "normalize-css": "~3.0.3",
-    "respond": "~1.4.2"
+    "respond": "~1.4.2",
+    "bootstrap-sass": "bootstrap-sass-official#~3.3.5"
```

Comment:
Yeah don't worry about this, this one slipped through... it isn't being used for anything quite yet. I'm dealing with the whole BS3 switchover in a separate branch